build/dist

```diff
@@ -270,6 +271,9 @@ echo "Build flags: $@" >> "$DISTDIR/RELEASE"
 # Copy kyuubi server jars
 cp -r "$KYUUBI_HOME"/kyuubi-assembly/target/scala-$SCALA_VERSION/jars/*.jar "$DISTDIR/jars/"
 
+# Copy kyuubi metadata sql
+cp -r "$KYUUBI_HOME"/kyuubi-assembly/target/kyuubi-assembly/metastore-sql/* "$DISTDIR/metastore-sql/"
```

Comment:
How about to directly copy `kyuubi-server/src/main/resources/sql` ?

We should not need `maven -assembly -plugin`.

Reply:
Considering the word "metastore" is famous as an alias of HMS, we'd better not use the same word in Kyuubi terms, I suggest use the generic words to name the folder, for example, "db-schema"